core/controllers/skill_editor.py

```diff
@@ -206,7 +252,7 @@ def put(self, skill_id):
         version = self.payload.get('version')
         _require_valid_version(version, skill.version)
 
-        commit_message = self.payload.get('commit_message')
+        commit_message = self.payload.get('commit_message', None)
```

Comment:
Here, would you rather want to have an empty string if no commit message was given or None is fine?

Reply:
You don't need to add `None` here but please change `payload` to `normalized_payload` ditto elsewhere in this handler.

Reply:
Done